test/fixtures/regional-dlp/versions.tf

```
@@ -31,12 +31,12 @@ terraform {
 
     null = {
       source  = "hashicorp/null"
-      version = "~> 2.1"
+      version = "3.2.0"
     }
 
     random = {
       source  = "hashicorp/random"
-      version = "~> 2.3"
+      version = "3.4.3"
```

Avoid needing to bump in the future

```suggestion
      version = ">= 2.3"
```